packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -227,7 +240,15 @@ export const warmupRequestHandler: Epic<AnyAction, AnyAction, any> = (action$, s
             }),
       ).pipe(
         mapTo(warmupFulfilled({ key })),
-        catchError((error: unknown) => of(warmupFailed({ error: error as Error, key }))),
+        catchError((error: unknown) =>
+          of(
+            warmupFailed({
+              id: getFeedId({ id: '0', data: requestData?.origin }),
```

Comment:
Should `id` always be 0?

Reply:
This is just a dummy ID to conform it to the expected Adapter Input type.

Reply:
Changed it to reflect warmer id